formula/src/main/java/com/instacart/formula/DeferredAction.kt

```diff
@@ -29,13 +30,13 @@ class DeferredAction<Event>(
 
         other as DeferredAction<*>
 
-        if (key != other.key) return false
+        if (initial != other.initial) return false
 
         return true
     }
```

Comment:
Does this change effect restarting an action?

Reply:
Each `key` has 1-1 mapping to the event listener. However, the listener provides identity comparison which performs much better than a data class equality.